src/js/components/Calendar/doc.js

```diff
@@ -95,12 +101,13 @@ to disable the previous and next buttons.
       For single select, make this the subsequent \`date\` property value.
       For multiple select or ranges, toggle values in \`dates\`.
       Not specifying this property makes the component read only.`),
-    range: PropTypes.bool
+    range: PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf(['array'])])
       .description(
         `Whether to automatically manage multiple date selection as a range.
         When the user clicks the first date, onSelect will be called with that
         date. When the user selects another date, onSelect will be called with
-        an array of two dates.`,
+        an array of two dates. If range = bounds, then an array of dates will 
```

Comment:
What does `bounds` mean in this context?

Reply:
Whoops, need to update the description. We've updated it to be `range="array"`.

Reply:
I thought so :)

Reply:
Should be updated now, let me know if the description makes sense.